README.md

```diff
@@ -1 +1,133 @@
-# OOP-school-libraray
\ No newline at end of file
+<a name="readme-top"><h1 align="center">OOP school library</h1></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 OOP School Library <a name="about-project"></a>
+
+OOP school library is a project that is part of the Microverse curriculum. The goal of this project is to practice the concepts of Object-Oriented Programming (OOP) in Ruby. The project is a console app that simulates a school library and it allows to perform the following actions:
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+- And all of this will be built in a beautiful and well-organized way!
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+- Ruby
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Create a app.rb file that will serve as your console app entry-point.
+- this will be a method called main that is invoked at the end of your file.
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+  install ruby
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git https://github.com/AbdifatahYasin1/OOP-school-libraray.git
+  cd school-library
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
```

Comment:
- [Optional] Since you are working alone, I think it's better to remove this header.